cmd/gaia/cli_test/test_helpers.go

```diff
@@ -234,19 +234,19 @@ func (f *Fixtures) TxBroadcast(fileName string, flags ...string) (bool, string,
 }
 
 //___________________________________________________________________________________
-// gaiacli tx stake
+// gaiacli tx staking
 
-// TxStakeCreateValidator is gaiacli tx stake create-validator
-func (f *Fixtures) TxStakeCreateValidator(from, consPubKey string, amount sdk.Coin, flags ...string) (bool, string, string) {
-	cmd := fmt.Sprintf("gaiacli tx stake create-validator %v --from=%s --pubkey=%s", f.Flags(), from, consPubKey)
+// TxStakingCreateValidator is gaiacli tx staking create-validator
```

Comment:
ditto with all the commands in this file

Reply:
see previous